tests/unit/UnknownBlockTest.php

```diff
@@ -26,49 +30,49 @@ public function setUp(): void
         <!-- wp:testing -->
         <p>Testing</p>
         <!-- /wp:testing -->
-      ')),
-      'post_status'  => 'publish'
-    ]);
-  }
+      '
+					)
+				),
+				'post_status'  => 'publish',
+			)
+		);
+	}
 
-  public function tearDown(): void
-  {
-    // your tear down methods here
-    parent::tearDown();
-    wp_delete_post($this->post_id, true);
-  }
+	public function tearDown(): void {
+		// your tear down methods here
+		parent::tearDown();
+		wp_delete_post( $this->post_id, true );
+	}
 
-  public function test_retrieve_content_blocks()
-  {
-    $query = '
+	public function test_retrieve_content_blocks() {
+		$query = '
 		{
 			posts(first: 1) {
 				nodes {
 					databaseId
           contentBlocks(flat: true) {
               name
-              parentId
               renderedHtml
           }
 				}
 			}
 		}
 		';
 
-    $actual = graphql(['query' => $query]);
-    $node = $actual['data']['posts']['nodes'][0];
+		$actual = graphql( array( 'query' => $query ) );
+		$node   = $actual['data']['posts']['nodes'][0];
+
+		// Verify that the ID of the first post matches the one we just created.
+		$this->assertEquals( $this->post_id, $node['databaseId'] );
 
-    // Verify that the ID of the first post matches the one we just created.
-    $this->assertEquals($this->post_id, $node['databaseId']);
+		// Verify 3 blocks are returned
+		$this->assertEquals( count( $node['contentBlocks'] ), 3 );
 
-    // Verify 3 blocks are returned
-    $this->assertEquals(count($node['contentBlocks']), 3);
+		$this->assertEquals( $node['contentBlocks'][0]['name'], 'core/paragraph' );
 
-    $this->assertEquals($node['contentBlocks'][0]['name'], 'core/paragraph');
+		$this->assertEquals( $node['contentBlocks'][1]['name'], 'core/heading' );
 
-    $this->assertEquals($node['contentBlocks'][1]['name'], 'core/heading');
-    
-    $this->assertEquals($node['contentBlocks'][2]['name'], 'UnknownBlock');
-    $this->assertEquals($node['contentBlocks'][2]['renderedHtml'], ' <p>Testing</p> ');
-  }
+		$this->assertEquals( $node['contentBlocks'][2]['name'], 'UnknownBlock' );
```

Comment:
For these tests, didn't contentBlocks get changed to editorBlocks via PR 1305?

Reply:
Its a separate ticket